materialize-bigquery/bigquery.go

```diff
@@ -31,6 +31,8 @@ type config struct {
 
 type advancedConfig struct {
 	UpdateDelay string `json:"updateDelay,omitempty" jsonschema:"title=Update Delay,description=Potentially reduce compute time by increasing the delay between updates. Defaults to 30 minutes if unset.,enum=0s,enum=15m,enum=30m,enum=1h,enum=2h,enum=4h"`
+
+	HardDelete bool `json:"hardDelete,omitempty" jsonschema:"title=Hard Delete,description=If this option is enabled items deleted in the source will also be deleted from the destination. By default is disabled and _meta/op in the destination will signify whether rows have been deleted (soft-delete).,default=false"`
```

Comment:
I'm wondering if this should be an advanced option, or if it should be a regular option. We've discussed and generally agreed that `UpdateDelay` should _not_ be an advanced option, but just haven't gotten around to re-working it to be in the regular section of the configuration. IMO hard deletes are something that should be in the regular configuration section, since we've seen people have trouble with the update delay config being in the advanced section.